pom.xml

```diff
@@ -759,7 +759,7 @@
             <dependency>
                 <groupId>mysql</groupId>
                 <artifactId>mysql-connector-java</artifactId>
-                <version>5.1.46</version>
+                <version>8.0.15</version>
```

Comment:
Since 8.0.18 has been released, when we finally merge this PR, can we bump it up to 8.0.18 please?